README.md

```diff
@@ -153,7 +153,21 @@ MarkerInfoWindow(
 }
 ```
 
-#### Obtaining Access to the raw GoogleMap (Experimental)
+### Street View
+
+You can a Street View given a location using the `StreetView` composable.
```

Comment:
missing verb. Did you mean "You can add"?

Reply:
Wups. Done